### PR TITLE
JCasC is now a multi-module project

### PR DESCRIPTION
**Pull request by ndeloof**
# Description
Additional JCasC modules for 0.10-alpha releaes
http://github.com/jenkinsci/configuration-as-code-plugin

# Submitter checklist for changing permissions

## Always

- [x] Add link to plugin/component Git repository in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### CODE REVIEW

**Comment by ndeloof** [comment]
cc @oleg-nenashev @batmat

**Comment by oleg-nenashev** [comment]
Looks like INFRA issue, CC @daniel-beck 

```
Djava.util.logging.SimpleFormatter.format=%1-%1-%1 %1:%1:%1 %4: %5%6%n -jar target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar
Running in dry run mode
Exception in thread "main" groovy.json.JsonException: Unable to process url: http://reports.jenkins.io/reports/artifactory-ldap-users-report.json
	at groovy.json.JsonSlurper.parseURL(JsonSlurper.java:416)
	at groovy.json.JsonSlurper.parse(JsonSlurper.java:379)
	at groovy.json.JsonSlurper$parse.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at io.jenkins.infra.repository_permissions_updater.ArtifactoryPermissionsUpdater.generateApiPayloads(ArtifactoryPermissionsUpdater.groovy:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrapNoCoerce.invoke(StaticMetaMethodSite.java:151)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:102)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:56)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:194)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:214)
	at io.jenkins.infra.repository_permissions_updater.ArtifactoryPermissionsUpdater.main(ArtifactoryPermissionsUpdater.groovy:317)
Caused by: java.io.FileNotFoundException: http://reports.jenkins.io/reports/artifactory-ldap-users-report.json
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at org.codehaus.groovy.runtime.ResourceGroovyMethods.configuredInputStream(ResourceGroovyMethods.java:2021)
	at org.codehaus.groovy.runtime.ResourceGroovyMethods.newReader(ResourceGroovyMethods.java:2070)
	at groovy.json.JsonSlurper.parseURL(JsonSlurper.java:410)
	... 17 more
```

**Comment by batmat** [comment]
It could be https://issues.jenkins-ci.org/browse/INFRA-1689 ? cc @olblak @daniel-beck

File: permissions/plugin-configuration-as-code-integrations.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "integrations"
+github: "jenkinsci/configuration-as-code-plugin"
+paths:
+- "io/jenkins/configuration-as-code/integrations"
```
Comment (batmat) [inline permissions/plugin-configuration-as-code-integrations.yml]: Now I'm wondering if the permission schema is recursive, and if so putting `io/jenkins/configuration-as-code` only wouldn't hence make things possibly simpler.

Comment (daniel-beck) [reply permissions/plugin-configuration-as-code-integrations.yml]: It is not.

Comment (daniel-beck) [inline permissions/plugin-configuration-as-code-integrations.yml]: This allows you to release the specific artifact called `integrations`. I doubt this is what's wanted here.

Comment (daniel-beck) [reply permissions/plugin-configuration-as-code-integrations.yml]: Oh, it is. Never mind then.

**Pull request merged**